packages/utils/interactive-hooks/src/useKeyProps.win32.ts

```diff
@@ -80,3 +80,22 @@ export const useKeyProps = memoize(getKeyUpPropsWorker);
 
 /** Exposes the behavior of useKeyProps for the current platform as a boolean */
 export const preferKeyDownForKeyEvents = false;
+
+/**
+ * Verifies if nativeEvent contains modifier key.
+ * @param nativeEvent
+ * @returns `true` if one or more of modifier keys are `true`
+ */
+export const isModifierKey = (nativeEvent: any): boolean => {
```

Comment:
It would be good to move this method to a common file. I see that some methods are repeated in win32 and windows files.

Reply:
Agreed, this could be re-used for macOS as well, as mentioned in another comment. I think we align on the type of the native event / names of modifier keys at least.

Reply:
Agreed, I think this makes sense to move to a new separate shared file